simple-backend/nlpviewer_backend/lib/stave_viewer.py

```diff
@@ -245,10 +268,93 @@ def open(self, url=None):
                 Default to None, which allows StaveViewer to automatically
                 set the url based on `in_viewer_mode`.
         """
-        webbrowser.open(url or (
-            f"{self.url}/viewer/0" if \
-            self.in_viewer_mode else self.url
-        ))
+        webbrowser.open(url or self.default_page)
+
+    @property
+    def default_page(self):
+        return f"{self.url}/viewer/0" if self.in_viewer_mode else self.url
+
+    @property
+    def db_file(self):
+        return self.load_config()[self.DB_FIELD]
+
+    @property
+    def log_file(self):
+        return self.load_config()[self.LOG_FIELD]
+
+    @classmethod
+    def set_config(cls, db_file: str = None, log_file: str = None):
+        """
+        Configure Stave.
+
+        Args:
+            db_file: Path to database. Default to None.
+            log_file: Path to log file. Default to None.
+        """
+        config = cls.load_config()
+        if db_file is not None:
+            config[cls.DB_FIELD] = db_file
+        if log_file is not None:
+            config[cls.LOG_FIELD] = log_file
+        with open(cls.CONFIG_FILE, "w") as f:
+            json.dump(config, f)
+
+    @classmethod
+    def load_config(cls):
+        """
+        Load or create configuration of Stave
+        """
+        if not os.path.isdir(cls.CONFIG_PATH):
+            os.mkdir(cls.CONFIG_PATH)
+
+        if not os.path.exists(cls.CONFIG_FILE):
+            with open(cls.CONFIG_FILE, "w") as f:
+                json.dump(cls.DEFAULT_CONFIG_JSON, f)
+
+        with open(cls.CONFIG_FILE, "r") as f:
+            return json.load(f)
+
+    def load_database(self,
+        load_auth: bool = False,
+        load_samples: bool = False
+    ):
+        """
+        Initialize sqlite database for django backend.
+
+        Args:
+            load_auth: Insert default user and authentication info to database.
+                Default to False.
+            load_samples: Indicate whether to load sample projects
+                into database. Deafult to False.
+        """
+        if not os.path.exists(self._db_file):
+            call_command("migrate")
+
+        if load_auth or load_samples:
+            User = get_user_model()
+            if not User.objects.filter(username=self.ADMIN_USERNAME).exists():
+                User.objects.create_superuser(
+                    self.ADMIN_USERNAME, '', self.ADMIN_PASSWORD)
```

Comment:
I think we can simplify a little y making the `ADMIN_USERNAME` to be created by default when create the database